source/isaaclab/isaaclab/utils/math.py

```diff
@@ -579,7 +603,6 @@ def quat_apply_yaw(quat: torch.Tensor, vec: torch.Tensor) -> torch.Tensor:
     return quat_apply(quat_yaw, vec)
 
 
-@torch.jit.script
 def quat_rotate(q: torch.Tensor, v: torch.Tensor) -> torch.Tensor:
     """Rotate a vector by a quaternion along the last dimension of q and v.
 
```

Comment:
Deprecation notice in docstrings was missing.

```suggestion
    .. deprecated v2.1.0:
         This function will be removed in a future release in favor of the faster implementation :meth:`quat_apply`.

```